packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -650,12 +670,11 @@ $ms-Persona-presenceSizeXl: 28px;
   .presence {
     height: $ms-Persona-presenceSizeXl;
     width: $ms-Persona-presenceSizeXl;
-    border-width: 4px;
   }
 
   .presenceIcon {
     line-height: $ms-Persona-presenceSizeXl;
-    font-size: $ms-font-size-xl;
+    font-size: $ms-font-size-m;
```

Comment:
Is this correct, considering the line-height is still XL?

Reply:
Yes, in this case, the line-height matches the presence height so that the font centers more naturally. I'll confirm with Matt and Ben that the icon's font-size is correct.